src/app/header/header.component.ts

```diff
@@ -1,19 +1,33 @@
 import { Component, OnInit } from '@angular/core';
 import {TranslateService} from '@ngx-translate/core';
+import {LocalstorageService} from '../core/localstorage.service';
 
 @Component({
   selector: 'ia-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
-  public selected = 'ru';
   constructor(public translate: TranslateService) {
     translate.addLangs(['en', 'ru']);
-    translate.setDefaultLang('ru');
   }
+  public selectedLang: any;
```

Comment:
1 - Don't use 'any' here, it must have string type
2 - If this selectedLang used only in one method - don't make it a class property, make it a local variable